apps/channels/urls.py

```diff
@@ -16,4 +16,5 @@
     ),
     path("whatsapp/turn/<uuid:experiment_id>/incoming_message", views.new_turn_message, name="new_turn_message"),
     path("api/<uuid:experiment_id>/incoming_message", views.new_api_message, name="new_api_message"),
+    path("commcare_connect/new_message/", views.new_connect_message, name="new_connect_message"),
```

Comment:
use consistent url paths
```suggestion
    path("commcare_connect/incoming_message", views.new_connect_message, name="new_connect_message"),
```